db/models.py

```diff
@@ -17,7 +21,7 @@ def __str__(self) -> str:
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
I guess this should be according to best practices in video, not as an argument to CharField.

(class Meta)

Reply:
But..... Create Meta class for 1 index it is creating of useless code. Maby in real project some space for future, but not here.

Reply:
Idk, I think it's better to always use best practices.